packager/media/codecs/video_slice_header_parser.cc

```diff
@@ -35,7 +35,7 @@ bool H264VideoSliceHeaderParser::Initialize(
     const Nalu& nalu = config.nalu(i);
     if (nalu.type() == Nalu::H264_SPS) {
       RCHECK(parser_.ParseSps(nalu, &id) == H264Parser::kOk);
-    } else {
+    } else if (nalu.type() == Nalu::H264_PPS) {
       DCHECK_EQ(Nalu::H264_PPS, nalu.type());
```

Comment:
Remove this check.

Reply:
Sorry for not making it clear. I mean removing the check in line 39, which is redundant. The comparison in line 38 is essential, otherwise SPS extension could be handled as PPS.